src/core/ConfigManager.cpp

```diff
@@ -260,18 +260,18 @@ void ConfigManager::setSTKDir(const QString & stkDir)
 
 
 
-void ConfigManager::setSF2Dir(const QString & sf2Dir)
+void ConfigManager::setSoundFontDir(const QString & soundFontDir)
```

Comment:
(Non-blocking) Per the Coding Conventions [re: pointers](https://github.com/LMMS/lmms/wiki/Coding-Conventions#pointers):

> The `*` symbol in pointer declarations MUST be adjacent to the type, not the variable name. This rule also applies to `**`, `&`.

```suggestion
void ConfigManager::setSoundFontDir(const QString& soundFontDir)
```